projects/power_distribution/src/front_uv_detector.c

```diff
@@ -0,0 +1,32 @@
+#include "can_transmit.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+
+static GpioAddress s_uv_comp_pin_address = { .port = GPIO_PORT_B, .pin = 0 };
```

Comment:
Instead of using this constant static variable, take (a pointer to) the UV comparator pin as input in `front_uv_detector_init`. You don't even need to store it statically, you can just pass it into `gpio_it_register_interrupt` and be done with it. In main.c, you can then use the `FRONT_UV_COMPARATOR_PIN` macro you defined to get the argument's value.